libs/orders/src/lib/components/order-list/order-view-dialog.tsx

```diff
@@ -0,0 +1,263 @@
+        <KeyValueTableRow key={'order-price'}>
+          <div data-testid={'order-price-label'}>{t('Price')}</div>
+          <div data-testid={`order-price-value`}>
+            {addDecimalsFormatNumber(
+              order.price,
+              order.market?.decimalPlaces as number
+            )}
+          </div>
+        </KeyValueTableRow>
+        <KeyValueTableRow key={'order-size'}>
+          <div data-testid={'order-size-label'}>{t('Size')}</div>
+          <div data-testid={`order-size-value`}>
+            <Size
+              value={order.size}
+              side={order.side}
+              positionDecimalPlaces={
+                order.market?.positionDecimalPlaces as number
+              }
+            />
+          </div>
+        </KeyValueTableRow>
+        <KeyValueTableRow key={'order-remaining'} className="mb-4">
+          <div data-testid={'order-remaining-label'}>{t('Remaining')}</div>
+          <div data-testid={`order-remaining-value`}>
+            <Size
+              value={order.remaining}
+              side={order.side}
+              positionDecimalPlaces={
+                order.market?.positionDecimalPlaces as number
+              }
+            />
+          </div>
+        </KeyValueTableRow>
+        <KeyValueTableRow key={'order-status'}>
+          <div data-testid={'order-status-label'}>{t('Status')}</div>
+          <div data-testid={`order-status-value`}>
+            {Schema.OrderStatusMapping[order.status as Schema.OrderStatus]}
+          </div>
+        </KeyValueTableRow>
+        {order.rejectionReason && (
+          <KeyValueTableRow key={'order-rejection-reason'}>
+            <div data-testid={'order-rejection-reason-label'}>
+              {t('rejection reason')}
+            </div>
+            <div data-testid={`order-rejection-reason-value`}>
+              {
+                Schema.OrderRejectionReasonMapping[
+                  order.rejectionReason as Schema.OrderRejectionReason
+                ]
+              }
+            </div>
+          </KeyValueTableRow>
+        )}
+        <KeyValueTableRow key={'order-id'}>
+          <div data-testid={'order-id-label'}>{t('Order ID')}</div>
+          <div data-testid={`order-id-value`}>
+            {truncateMiddle(order.id, 10)}
+            <CopyToClipboard text={order.id} onCopy={() => setCopied(true)}>
+              <button
+                type="button"
+                data-testid="copy-vega-public-key"
+                onClick={(e) => e.stopPropagation()}
+              >
+                <span className="sr-only">{t('Copy')}</span>
+                <VegaIcon name={VegaIconNames.COPY} />
+              </button>
+            </CopyToClipboard>
+          </div>
+        </KeyValueTableRow>
+        <KeyValueTableRow key={'order-created'}>
+          <div data-testid={'order-created-label'}>{t('Created')}</div>
+          <div data-testid={`order-created-value`}>
+            {new Date(order.createdAt).toLocaleString()}
```

Comment:
Why not use our Intl object?

`getDateTimeFormat().format(...)`

Reply:
updated